packages/validators/src/conditions/date-condition.ts

```
@@ -0,0 +1,38 @@
+import { z } from "zod";
+
+const operator = z.union([
+  z.literal("before"),
+  z.literal("after"),
+  z.literal("before-or-on"),
+  z.literal("after-or-on"),
+]);
+
+const value = z.string().refine((v) => !isNaN(new Date(v).getTime()), {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)